torch/manage_context.py

```diff
@@ -0,0 +1,29 @@
+"""
+Manage various cudnn contexts
+"""
+
+import torch
+from contextlib import contextmanager
+
+
+@contextmanager
+def cudnn_context(use_cudnn=False, benchmark=False, deterministic=False, verbose=False):
```

Comment:
This should go to `torch.backends.cudnn` (somewhere close to where these flags are defined). I don't think we need a separate file for context managers. Let's keep them close to what they are needed for. I'd imagine it used like this:
```python
from torch.backends import cudnn

with cudnn.flags(enabled=True, benchmarks=False, deterministic=True):
    ...
```

Also, please use `enabled` instead of `use_cudnn`, so it matches the flag.

Reply:
I was hoping that we could use this manage_context file as a general file to have context for other things as well as we come across use cases in future. I can move it to cudnn itself though if we don't see much utility for it yet

Reply:
We already have a few context manager, but we keep them in their respective modules. Let's keep this convention. I think it makes sense

Reply:
done! this should be mergeable now after waiting for contbuild